src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/unit/domain/entity/factory/AudienceEntityFactory.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.unit.domain.entity.factory
+
+import io.github.bluegroundltd.kfactory.Factory
+import io.github.bluegroundltd.kfactory.Yielded
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.common.util.randomLowercaseString
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.domain.entity.create.AudienceEntity
+import java.util.*
+
+class AudienceEntityFactory : Factory<AudienceEntity> {
+  private var id: Yielded<UUID?> = { UUID.randomUUID() }
+  private var value: Yielded<String> = { randomLowercaseString() }
+
+  fun withId(id: UUID?) = apply {
```

Comment:
doesn't appear to be used can it be removed if not?

Reply:
Done, and done for other unused methods in each factory 👍